sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapRfRExamples.cs

```diff
@@ -0,0 +1,195 @@
+            var upsertResponse = _instrumentsApi.UpsertInstruments(definitions);
+            ValidateUpsertInstrumentResponse(upsertResponse);
+
+            // CAN NOW QUERY FROM LUSID
+            var getResponse = _instrumentsApi.GetInstruments("ClientInternal", new List<string> { uniqueId });
+            ValidateInstrumentResponse(getResponse, uniqueId);
+            
+            var retrieved = getResponse.Values.First().Value.InstrumentDefinition;
+            Assert.That(retrieved.InstrumentType == LusidInstrument.InstrumentTypeEnum.InterestRateSwap);
+            var roundTripSwap = retrieved as InterestRateSwap;
+            Assert.That(roundTripSwap, Is.Not.Null);
+            Assert.That(roundTripSwap.MaturityDate, Is.EqualTo(swap.MaturityDate));
+            Assert.That(roundTripSwap.StartDate, Is.EqualTo(swap.StartDate));
+            Assert.That(roundTripSwap.Legs.Count, Is.EqualTo(swap.Legs.Count));            
+            
+            // DELETE instrument
+            _instrumentsApi.DeleteInstrument("ClientInternal", uniqueId);
+        }
+       
+        [LusidFeature("F22-47")]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney, "SOFR")]
+        [TestCase(ModelSelection.ModelEnum.Discounting, "SOFR")]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney, "ESTR")]
+        [TestCase(ModelSelection.ModelEnum.Discounting, "ESTR")]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney, "SONIA")]
+        [TestCase(ModelSelection.ModelEnum.Discounting, "SONIA")]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney, "TONA")]
+        [TestCase(ModelSelection.ModelEnum.Discounting, "TONA")]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney, "SARON")]
+        [TestCase(ModelSelection.ModelEnum.Discounting, "SARON")]
+        public void InterestRateSwapRfRInlineValuationExample(ModelSelection.ModelEnum model, string rfr)
+        {
+            var irs = InstrumentExamples.CreateExampleInterestRateSwap(rfr);
+            CallLusidInlineValuationEndpoint(irs, model);
+        }
+        
+        [LusidFeature("F22-48")]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney, "SOFR")]
+        [TestCase(ModelSelection.ModelEnum.Discounting, "SOFR")]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney, "ESTR")]
+        [TestCase(ModelSelection.ModelEnum.Discounting, "ESTR")]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney, "SONIA")]
+        [TestCase(ModelSelection.ModelEnum.Discounting, "SONIA")]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney, "TONA")]
+        [TestCase(ModelSelection.ModelEnum.Discounting, "TONA")]
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney, "SARON")]
+        [TestCase(ModelSelection.ModelEnum.Discounting, "SARON")]
+        public void InterestRateSwapRfRPortfolioCashFlowsExample(ModelSelection.ModelEnum model, string rfr)
```

Comment:
"rfr" doesn't mean anything to me. Maybe a more verbose name.

Reply:
RFR is quite a common term like LIBOR - don't think it needs expanding in this context.